### DISCUSSION TITLE
Missing icons

### DISCUSSION THREAD

**Discussion by bengravesdev** [malzariey/filament-daterangepicker-filter#146] [Q&A] [answered]
Do I have to do something specific to make the calendar icon display?

I'm using the picker and filter in several places but the icon isn't showing and I can't clear the selected values.

I'm using Filament 3.2 and daterangepicker 3.1

**Comment by malzariey** [comment, marked as answer]
Did you try this

```DateRangeFilter::make('created_at')->disableClear(false)```
